server/utils/bedUtils.test.ts

```diff
@@ -40,55 +52,72 @@ describe('bedUtils', () => {
     })
   })
 
-  describe('roomNameCell', () => {
-    it('returns the name of the room', () => {
-      expect(roomNameCell(bedSummary)).toEqual({ text: bedSummary.roomName })
+  describe('bedsTableRows', () => {
+    const bedList = cas1PremisesBedSummaryFactory
+      .buildList(5)
+      .sort(({ bedName: name1 }, { bedName: name2 }) =>
+        name1.localeCompare(name2, 'en', { numeric: true, sensitivity: 'base' }),
+      )
+
+    it('returns the rows of the beds table', () => {
+      const expected = bedList.map(bed => {
+        return [
+          htmlCell(
+            `<a href="/manage/premises/premisesId/beds/${bed.id}" data-cy-bedId="${bed.id}">${bed.bedName}<span class="govuk-visually-hidden">bed ${bed.bedName}</span></a>`,
```

Comment:
Screen readers will read the main text content _and_ the hidden text, which means the content here is duplicated, as it is read as, for example, `1-1 bed 1-1` (this has been pointed out as a medium issue elsewhere in the accessibility audit). Ideally the `linkTo` utility should allow a hidden prefix _and/or_ suffix, so the hidden text could be `Bed name:` before the content. Or the `hiddenText` property could be ignored and the hidden element simply passed as part of the `text` argument? 

As an aside, after a quick look I'm not sure the `hiddenText` property is very judiciously used in many places! ☹️